### ISSUE TITLE
Python3 is in SLE15SP4

### ISSUE THREAD

**Issue by dominicpg** [uyuni-project/uyuni-docs#1649]
Installation https://documentation.suse.com/suma/4.3/en/suse-manager/installation-and-upgrade/install-vm.html 
Step (4) under "Preparing Minimal SLES for SUSE Manager" needs to be updated by Python3. 

Current 

SUSEConnect -p sle-module-python2/15.4/x86_64

SLES15SP4 has Python3 and this needs to be corrected accordingly.

**Comment by mcalmer** [comment]
@keichwa this is right. Python2 module is not needed an more for 4.3. The python3 module is optional!!!

**Comment by mcalmer** [comment]
AFAIK python3 module has only python3.10 with only a few modules. The main python version in SLE15 is python3.6 which is in the basesystem module.
With SUMA there is no reason for enabling python3 module. On a new installation we don't do it.
If you need to install extra software on the system which requires python 3 > 3.6 than you may want it.

**Comment by keichwa** [comment]
Thanks for all your background info!  In this case, I agree that we can simply remove sle-module-pythonX.  See https://github.com/uyuni-project/uyuni-docs/pull/1650 now.

**Comment by juliogonzalez** [comment]
@keichwa this could be handy in the future:

https://maintenance.suse.de/reports/graphs/products (only available for SUSE employees)

If you check, for example, SUSE Manager 4.3 Server x86_64, you will see that the python3 module is optional (black arrow), while the mandatory modules have the arrows in red

**Comment by keichwa** [comment]
Very useful, but font is very small.  Workaround: mouse-over.
